internal/impl/devbox.go

```diff
@@ -726,7 +727,45 @@ func (d *Devbox) installNixProfile() (err error) {
 		}
 
 		cmd.Env = nix.DefaultEnv()
-		_, err = cmd.Output()
+
+		// Get a pipe to read from standard out
+		r, _ := cmd.StdoutPipe()
```

Comment:
I think this is an error we probably want to handle?

Reply:
good call